sql/catalyst/benchmarks/GenericArrayDataBenchmark-results.txt

```diff
@@ -1,10 +1,10 @@
-OpenJDK 64-Bit Server VM 1.8.0_362-b09 on Linux 5.15.0-1031-azure
-Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz
+OpenJDK 64-Bit Server VM 17.0.8+7-LTS on Linux 5.15.0-1046-azure
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 constructor:                              Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-arrayOfAny                                            4              4           0       2491.5           0.4       1.0X
-arrayOfAnyAsObject                                  256            257           1         39.1          25.6       0.0X
-arrayOfAnyAsSeq                                      18             18           0        551.9           1.8       0.2X
-arrayOfInt                                          536            537           1         18.7          53.6       0.0X
-arrayOfIntAsObject                                  788            794          10         12.7          78.8       0.0X
+arrayOfAny                                            7              7           0       1495.4           0.7       1.0X
+arrayOfAnyAsObject                                    7              7           0       1495.3           0.7       1.0X
```

Comment:
The results of `arrayOfAnyAsSeq` have undergone significant changes and need attention (it may be a known issue, but I can't remember the details).